addons/l10n_in/models/base_document_layout.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    country_code = fields.Char(related="company_id.account_fiscal_country_id.code")
```

Comment:
Add this in `account` model directly so it's help full for other localization.
right now other localization already add same fields

Here is some examples 
https://github.com/odoo/odoo/blob/57573994313988837d89329d77ab1def63a8cfdd/addons/l10n_ca/models/res_company.py#L14
https://github.com/odoo/odoo/blob/57573994313988837d89329d77ab1def63a8cfdd/addons/l10n_cz/models/res_company.py#L19
https://github.com/odoo/odoo/blob/57573994313988837d89329d77ab1def63a8cfdd/addons/l10n_din5008/models/base_document_layout.py#L38

**so remove this all and add one in account**

Reply:
@hamo-odoo so it means that this all defined fields `account_fiscal_country_id` in `base.document.layout` are dead code?

Reply:
Oh, Here in report preview company is not `res.company` but `base.document.layout`
https://github.com/odoo/odoo/blob/7faddab81673568f9a5e20107957f521c59247ca/addons/web/models/base_document_layout.py#L134-L141

So in another PR we define `account_fiscal_country_id` in `base.document.layout` form account module and remove it from all depended modules